### PR TITLE
feat(prejoin) allow hiding prejoin display name edit field

### PR DESCRIPTION
**Pull request by quitrk**
Screenshot 2022-05-24 at 12 42 26

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -495,6 +495,9 @@ var config = {
     // Hides add breakout room button
     // hideAddRoomButton: false,
 
+    // Require users to always specify a display name.
```
Comment (shawnchin) [inline config.js:498]: This config could use a better description.

*(P.S. Thanks for this! Much cleaner than just disabling the input field using readOnlyName option.)*

Comment (quitrk) [reply config.js:498]: updated, thank you

**Review by horymury — APPROVED** [review]

**Review by horymury — APPROVED** [review]

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by quitrk** [comment]
Jenkins please test this please

**Comment by jluisfa** [comment]
In meet.jit.si the avatar does not appear centere

imagem
d

**Comment by jluisfa** [comment]
At alpha.jitsi.net the behavior is the same.
imagem1
I have marked with a red arrow where the problem is.
"class" and "style" are empty `<div class style>`

If you put `<div class style="text-align:center;">` the problem is fixed

imagem2

**Comment by saghul** [comment]
Ping @quitrk PTAL when you're back.

**Pull request merged**